packages/debug/src/common/debug-configuration.ts

```diff
@@ -84,6 +84,9 @@ export interface DebugConfiguration {
 
     /** When true, the debug viewlet will not be automatically revealed for this session. */
     suppressDebugView?: boolean;
+
+    /** Disable the warning when trying to start the same debug configuration more than once. `true` by default. */
+    suppressMultipleSessionWarning?: boolean;
```

Comment:
Note that the current PR behaves as if `false` is default, i.e. it will warn when the option is not given

Reply:
Fixed via https://github.com/eclipse-theia/theia/pull/14862/commits/df5ec4dd2ac57a5e7c4a6940210b02a8c0278509.